styleguide/_themes/derek/scss/snowflakes/events.scss

```diff
@@ -0,0 +1,364 @@
+.events-banner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/solve-banner.jpg');
+}
+
+// Main Events Page.
+.eventsOverview {
+  @include make-container;
+}
+
+.eventsOverview-pagination {
+  list-style-type: none;
+  margin: 0;
+  padding: 2rem 0 0;
+  text-align: center;
+}
+
+.eventsOverview-pagination-item {
+  display: inline-block;
+  list-style-type: none;
+}
+
+.eventsOverview-pagination-itemPage {
+  background: $white;
+  border: 1px solid $gray-light;
+  font-size: 1rem;
+  padding: .5rem;
+  text-decoration: none;
+
+  &:hover {
+    background: $gray-lightest;
+    color: $black;
+    text-decoration: none;
+  }
+}
+
+.eventsOverview-pagination-itemPageActive {
+  font-size: 1.4rem;
+}
+
+.eventsOverview-pagination-itemPagePrev,
+.eventsOverview-pagination-itemPageNext {
+  @include border-radius(50%);
+  background: $teal-dark;
+  border: 0;
+  color: $white;
+  padding: 0 .5rem;
+
+  &:hover {
+    background: $gray-darkest;
+    color: $white;
+  }
+}
+
+.eventsOverview-row {
+  @include make-row;
+}
+
+.eventsOverview-searchFilter {
+  @include make-sm-column(4);
+  @include make-md-column(3);
+  background: $teal-dark;
+  margin-bottom: 1rem;
+  padding-bottom: 1rem;
+  padding-top: 1rem;
+}
+
+.eventsOverview-searchFilter-label {
+  color: $white;
+  font-size: 2.5rem;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.eventsOverview-searchFilter-eventTypeSelect {
+  background: $white;
+  display: block;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  width: $full-width;
+}
+
+.eventsOverview-searchFilter-eventTypeSubmit {
+  background: $gray-base;
+  border: 0;
+  color: $white;
+  display: block;
+  font-size: 1.2rem;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  text-transform: uppercase;
+  width: $full-width;
+
+  &:hover {
+    background: $gray-dark;
+  }
+}
+
+.eventsOverview-searchResults {
+  @include make-sm-column(8);
+  @include make-md-column(9);
+  margin-bottom: 3rem;
+}
+
+.eventsOverview-weekEvent {
+  @include make-row;
+  background: $white;
+  border: 1px solid $gray-light;
+  margin: 0 0 .75rem;
+  text-align: center;
+}
+
+.eventsOverview-weekEvent-date {
+  background: $teal-dark;
+  color: $white;
+  font-weight: 600;
+  line-height: 3rem;
+  padding: 0 1.5rem;
+  text-transform: uppercase;
+}
+
+.eventsOverview-weekEvent-details {
+  padding: .5rem 1.5rem 1rem;
+}
+
+.eventsOverview-weekEvent-header {
+  @include make-row;
+}
+
+.eventsOverview-weekEvent-headerLocation {
+  @include make-xs-column(12);
+  @include make-md-column(5);
+  font-size: 1.2rem;
+  line-height: 4rem;
+  padding: 0 3rem 0 1.5rem;
+  text-transform: uppercase;
```

Comment:
Yeah I'll lump that into the font refactor task. I think we need to take a stab at more reusability around these sorts of things for fonts since we likely wont be in title case forever. But you're right I dont think this is the place to tackle it.